### PR TITLE
Fix safari/ios 13

### PR DESCRIPTION
**Pull request by nickgal**
Fixes regression from https://github.com/aeharding/wefwef/pull/74

Application fails to load (blank page) with error `mediaMatch().addEventListener is not a function`.

See also: https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList#browser_compatibility

### CODE REVIEW

**Comment by aeharding** [comment]
Thanks for this!

Can you please write a short inline comment on why, and/or linking to this PR? (just write "#264")

Just so it doesn't accidentally get "simplified" later 😅

**Comment by nickgal** [comment]
Does that work?

**Comment by aeharding** [comment]
Looks good, but prettier is complaining

File: src/helpers/useSystemDarkMode.ts [hunk 1]
```diff
@@ -7,22 +7,30 @@ export default function useSystemDarkMode() {
     window.matchMedia(DARK_MEDIA_SELECTOR).matches
   );
 
+  const mediaQuery = window.matchMedia(DARK_MEDIA_SELECTOR);
```
Comment (aeharding) [inline src/helpers/useSystemDarkMode.ts:10]: Why was this moved out of the effect?

**Pull request merged**